packages/web-components/src/components/rux-tooltip/rux-tooltip.tsx

```diff
@@ -264,9 +263,8 @@ export class RuxTooltip {
                         ref={(el) => (this.trigger = el!)}
                         aria-describedby="tooltip"
                     >
-                        <slot onSlotchange={_handleSlotChange} />
+                        <slot onSlotchange={_handleSlotChange}></slot>
                     </span>
-                    <Fragment>&#8203;</Fragment>
```

Comment:
I'm not sure what this was here for but it does seem to be the problem child.

Reply:
I'm pretty sure this was to solve this issue:

```
whatup its <rux-tooltip message="mark">yer</rux-tooltip> boi
```
![Screenshot 2023-08-16 at 4 42 46 PM](https://github.com/RocketCommunicationsInc/astro/assets/1091342/fb6ba9c5-f518-43d9-b148-90cf920ce431)

If I double click on "yer" to select the text, the tooltip text should not be selected. 
https://codepen.io/rocketmark/pen/eYbOrab

but in your PR with this out, this is what you get. 
![Screenshot 2023-08-16 at 4 43 32 PM](https://github.com/RocketCommunicationsInc/astro/assets/1091342/84785760-c4e1-4802-acc6-ec66d150c03e)

Reply:
Oh good that helps! I'll see if I can solve for both in the morning. Thanks for the context!!

Reply:
@markacianfrani I'm pretty sure that we don't want people to accidentally select the tooltip at all, so what if, as an alternate solution, we just add the css `user-select` (and `-webkit-user-select`) `none`?

I've updated the PR with this. this should make it so that not only does it not select the tooltip when double clicking but it also won't select it if you drag to select the whole phrase.